### DISCUSSION TITLE
What's the use case for docusaurus in the gemfile package?

### DISCUSSION THREAD

**Discussion by RailsCod3rFuture** [activerecord-hackery/ransack#1360] [Q&A] [answered]
Docusaurus is showing up as a false positive in AWS. Is it mandatory to store it within the gem build folder in docs/?

**Comment by deivid-rodriguez** [comment]
Hello @RailsCod3rFuture!

Can you elaborate a bit more? Which kind of false positive are you getting? We do ship the docs folder as part of the gem, but I don't see much issues with doing that?

**Comment by scarroll32** [comment, marked as answer]
https://github.com/activerecord-hackery/ransack/issues/1680
